omisego-sdk/consumer-proguard-rules.pro

```diff
@@ -0,0 +1,25 @@
+# OMG
+-keep class co.omisego.omisego.** { *; }
+-keep interface co.omisego.omisego.** { *; }
+-keep class * implements co.omisego.omisego.** { *; }
+
+# Save the obfuscation mapping to a file, so we can de-obfuscate any stack
+# traces later on. Keep a fixed source file attribute and all line number
+# tables to get line numbers in the stack traces.
+-printmapping out.map
+-keepparameternames
+-renamesourcefileattribute SourceFile
+-keepattributes Exceptions,InnerClasses,Signature,Deprecated,SourceFile,LineNumberTable,EnclosingMethod
+
+# Gson
+-keepattributes Signature
+-keepattributes *Annotation*
+-keepattributes EnclosingMethod
+
+# OkHttp
```

Comment:
Rules seems overkill, take a look at https://github.com/square/okhttp#proguard this should be enough?

Reply:
Fixed